pkgs/development/libraries/libvterm-neovim/default.nix

```diff
@@ -17,8 +17,11 @@ stdenv.mkDerivation rec {
 
   nativeBuildInputs = [ perl libtool ];
 
-  makeFlags = [ "PREFIX=$(out)" ]
-    ++ lib.optional stdenv.isDarwin "LIBTOOL=${libtool}/bin/libtool";
+  makeFlags = [
+    "CC=${stdenv.cc.targetPrefix}cc"
+    "LIBTOOL=${libtool}/bin/libtool"
```

Comment:
Turns out this is also useful for cross compilation, since its does not harm native compilation, lets just have enabled by default.